### PR TITLE
Fix EF migrations

### PR DESCRIPTION
**Pull request by CodingVelocista**

### CODE REVIEW

File: embc-app/Migrations/20190506213507_EvacueeRegistration-Evacuee-EvacueeAddress-Tables.cs [hunk 1]
```diff
@@ -92,7 +92,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
                         column: x => x.CommunityId,
                         principalTable: "Communities",
                         principalColumn: "Id",
-                        onDelete: ReferentialAction.Cascade);
+                        onDelete: ReferentialAction.Restrict);
```
Comment (ytqsl) [inline embc-app/Migrations/20190506213507_EvacueeRegistration-Evacuee-EvacueeAddress-Tables.cs]: restrict can be dangerous, maybe DeleteBehavior.ClientSetNull?

File: embc-app/Migrations/20190426215123_Country-PK-Change.cs [hunk 2]
```diff
@@ -284,6 +284,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
                                                 UPDATE Countries SET CountryCode = 'VNM' WHERE Name = 'Viet Nam'
                                                 UPDATE Countries SET Name = 'Virgin Islands (British)' WHERE Name = 'Virgin Islands, British'
                                                 UPDATE Countries SET CountryCode = 'VGB' WHERE Name = 'Virgin Islands (British)'
+                                                UPDATE Countries SET Name = 'Virgin Islands (U.S.)' WHERE Name = 'Virgin Islands, U.S.'
```
Comment (ytqsl) [inline embc-app/Migrations/20190426215123_Country-PK-Change.cs]: I'd prefer to generate the update from the old json file to ensure the names are the same

Comment (CodingVelocista) [reply embc-app/Migrations/20190426215123_Country-PK-Change.cs]: Me as well.  Just couldn't figure out a clean way to inject the DataInterface.

File: embc-app/Migrations/20190506213507_EvacueeRegistration-Evacuee-EvacueeAddress-Tables.cs (hunk 1 above)
Comment (CodingVelocista) [reply embc-app/Migrations/20190506213507_EvacueeRegistration-Evacuee-EvacueeAddress-Tables.cs]: Yes that would be safer.  I'll change it in 'Develop' and push it with the next Master merge.

**Comment by ytqsl** [comment]
Will it work if the migration was already applied?

**Pull request merged**